package.json

```diff
@@ -267,6 +267,7 @@
     "@react-stately/tree": "3.2.0",
     "@react-types/menu": "3.4.1",
     "@react-types/shared": "^3.10.1",
+    "@reactour/tour": "^2.10.3",
```

Comment:
It's amazing! I didn't know about existing such cool libraries. Generally, I have the same question as @filipmikes1 about text. It seems to me it will looks better if we can move current steps into different file and then import like step1 or stepName into array, so the array will be shorter and it will be easier to read this code in future

Reply:
Absolutely!